### PR TITLE
ipmitool: upgrade version 1.8.18 -> 1.8.19

### PR DESCRIPTION
**Pull request by mfrw**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Upgrade version 1.8.18 -> 1.8.19

###### Change Log  

- Upgrade version 1.8.18 -> 1.8.19
- Add OEM enterprise dictionary [reference](https://github.com/ipmitool/ipmitool/commit/bd0475ce)

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

- Pipeline build id: [PR-7176](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=479768&view=results) ❌ (pipeline has issues resolving systemd)
Local Build:
image

### CODE REVIEW

File: SPECS/ipmitool/enterprise-numbers
Comment (rlmenge) [inline SPECS/ipmitool/enterprise-numbers:1]: Might be worth mentioning in PR description that this file was needed to work with https://github.com/ipmitool/ipmitool/commit/bd0475ce4ad85645f893010423b265ada1514cc5  and avoid downloading

**Review by rlmenge — COMMENTED** [review]
Changes LGTM. Will approve when pipeline and install are complete

File: SPECS/ipmitool/enterprise-numbers
Comment (mfrw) [reply SPECS/ipmitool/enterprise-numbers:1]: added a changelog entry and updated the PR description as well.

**Comment by mfrw** [comment]
> Changes LGTM. Will approve when pipeline and install are complete

The BuddyBuild has (known) issues resolving some packages. Locally i ensured the build is passing.

**Review by rlmenge — APPROVED** [review]

**Pull request merged**